src/main/java/bio/terra/app/utils/SentryUtils.java

```diff
@@ -0,0 +1,46 @@
+package bio.terra.app.utils;
+
+import bio.terra.app.configuration.SentryConfiguration;
+import bio.terra.common.exception.BadRequestException;
+import bio.terra.common.exception.ConflictException;
+import bio.terra.common.exception.ForbiddenException;
+import bio.terra.common.exception.NotFoundException;
+import bio.terra.common.exception.NotImplementedException;
+import io.sentry.Sentry;
+import java.util.Optional;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.servlet.NoHandlerFoundException;
+
+public class SentryUtils {
+  private static final String DEFAULT_UNDEFINED_ENVIRONMENT = "undefined";
+
+  public static void initializeSentry(SentryConfiguration sentryConfiguration) {
+    String dsn = Optional.ofNullable(sentryConfiguration.getDsn()).orElse("");
+    String environment =
+        Optional.ofNullable(sentryConfiguration.getEnvironment())
+            .orElse(DEFAULT_UNDEFINED_ENVIRONMENT);
+    // in order to filter out exceptions, we must initialize sentry options
+    // otherwise, they can be automatically configured via application.properties
+    Sentry.init(
+        options -> {
+          options.setDsn(dsn);
+          options.setEnvironment(environment);
+          // Filter out exceptions we don't want to send to Sentry
+          options.setBeforeSend(
+              (event, hint) -> {
+                Throwable throwable = event.getThrowable();
+                if (throwable instanceof NotFoundException
```

Comment:
I think making this code data driven would improve its readability. One way to do this:
```
  private static final List<Class<? extends Exception>> FILTERED = List.of(
    NotFoundException.class,
    BadRequestException.class,
    NotImplementedException.class,
    ConflictException.class,
    MethodArgumentNotValidException.class,
    IllegalArgumentException.class,
    NoHandlerFoundException.class,
    ForbiddenException.class
  );
...
          options.setBeforeSend(
              (event, hint) -> {
                Throwable throwable = event.getThrowable();
                if (FILTERED.stream().anyMatch(e -> e.isInstance(throwable))) {
                  return null;
                }
                return event;
              });
```

Reply:
Thanks for taking a look at this, this is much better.